src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -1828,4 +1831,26 @@ private String getExtensionEventText(MultiPartyScenario scenario, PartyData part
                 return format("agreed extension date: %s", extensionDate);
         }
     }
+
+    //How are we capturing the reason that it is dropped offline?
+    private void buildSDONotDrawn(EventHistory.EventHistoryBuilder builder,
+                                  CaseData caseData) {
+        if (featureToggleService.isSDOEnabled()) {
```

Comment:
@sampankumar I have split the task up into two, so I check the toggle here in the 'buildSDONotDrawn' method.
Should I have it in the 'FullDefenseProceed' as well as per your comment?